vendor/modules.txt

```diff
@@ -1582,7 +1582,6 @@ k8s.io/api/rbac/v1beta1
 k8s.io/api/scheduling/v1
 k8s.io/api/scheduling/v1alpha1
 k8s.io/api/scheduling/v1beta1
-k8s.io/api/settings/v1alpha1
```

Comment:
looks like a few other packages need cleaning up from this file, hack/update-vendor.sh should resolve it